### DISCUSSION TITLE
Modifying SQL used for data viewer

### DISCUSSION THREAD

**Discussion by lavalex** [dbeaver/dbeaver#15278] [Q&A]
Hello,

Is it possible to modify underlying SQL that is used for data view? For example, I using a JDBC driver that is not out of the box and to view data in the table, I need to specify name of the database and then name of the table. DBeaver only specified table name and it doesn't work.

Thank you.

**Comment by LonwoLonwo** [comment]
If you added your driver as a Generic driver, then you can find some custom settings in the "Connection Settings" -> "Edit Driver Settings" -> "Advanced parameters". Maybe they can help you.

![2022-01-25 13_14_27-DBeaver Ultimate 21 3 3 - testpass2@%](https://user-images.githubusercontent.com/45152336/150957757-a30466e1-98b7-40b9-9594-923ec9dc62ce.png)

**Comment by lavalex** [comment]
Thank you, I checked, there is nothing that seems to be related to data viewer.
